ads/google/a4a/docs/RTBExchangeGuide.md

```diff
@@ -1,82 +1,82 @@
 # AMPHTML Ad Implementation Guide for RTB Ad Exchanges
 ---
 ## Objective
- 
```

Comment:
I believe trailing spaces have a special purpose in `.md` files, and shouldn't automatically be removed.

I assume your editor did this. There's a way to disable the behavior for vscode: https://github.com/Microsoft/vscode/issues/1679

Can you revert these changes?

Reply:
@rsimha I believe this issue is resolved, can you confirm?

Reply:
It isn't. See https://github.com/ampproject/amphtml/pull/22126/files#diff-12cfb8e62838a6be57d7f1585dd104ac.